packages/core/src/generatePolicy.js

```diff
@@ -73,9 +90,10 @@ function createModuleInspector(opts) {
    */
   function inspectModule(
     moduleRecord,
-    { isBuiltin, includeDebugInfo = false }
+    { isBuiltin, includeDebugInfo = false, trustRoot = true }
   ) {
     if (moduleRecord === undefined) {
+      // XXX: when does this happen?
```

Comment:
https://github.com/LavaMoat/LavaMoat/commit/7599146f6412777dd13d681283e1b5344297d0eb
Introduced when I enabled going through overrides in classic lavamoat

Reply:
- [ ] update comment